packages/ckeditor5-image/src/imagestyle/imagestyleediting.js

```diff
@@ -31,36 +31,59 @@ export default class ImageStyleEditing extends Plugin {
 	 */
 	init() {
 		const editor = this.editor;
-		const schema = editor.model.schema;
-		const data = editor.data;
-		const editing = editor.editing;
+		const isBlockPluginLoaded = editor.plugins.has( 'ImageBlockEditing' );
+		const isInlinePluginLoaded = editor.plugins.has( 'ImageInlineEditing' );
+
+		editor.config.define( 'image.styles', getDefaultStylesConfiguration( isBlockPluginLoaded, isInlinePluginLoaded ) );
 
-		// Define default configuration.
-		editor.config.define( 'image.styles', [ 'full', 'side' ] );
+		/**
+		 * TODO docs
+		 *
+		 * Clear the arrangements and groups from the unsupported and undefined items.
+		 */
+		this.normalizedStyles = normalizeStyles( {
```

Comment:
Typedefs for arrangements and groups are needed, similar to `module:image/imagestyle/imagestyleediting~ImageStyleFormat`.

Reply:
Done.